wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractWComponent.java

```diff
@@ -166,8 +155,8 @@ public void setIdName(final String idName) {
 	}
 
 	/**
-	 * {@inheritDoc}
-	 *
+         * {@inheritDoc}
```

Comment:
Can you explain this change of indent character? All Java is indented with tabs as per the contributing guidelines. Please re-set to the initial code lines.